Added support for Solidity 'import' statements

Antanukas opened the pull request:
- Using solc compile function which calls a callback when import statement gets detected. This function is responsible for loading file source.
- Upgraded solc version to latest just in case.

Commit:
Added support for Solidity 'import' statements, upgraded solc dep version

uzyn commented:
@Antanukas Thanks. Can you link to a solidity code that we could test this new feature?

On a separate note, we're going to need unit tests and better management on the loaders soon. (https://github.com/uzyn/ethereum-webpack-example-dapp/pull/3 is a good start)

Antanukas commented:
I have updated example-dapp with additional contract in external file which gets imported. Also added dummy library that gets imported. https://github.com/Antanukas/ethereum-webpack-example-dapp

This also tests [web3-loader](https://github.com/uzyn/web3-loader/pull/16)

graup commented:
Thanks @Antanukas, just what I needed now! You just saved me a lot of time.

This works for me, let's get it merged!

Commit:
Bump solc version
